doc/OSAL-Configuration-Guide.md

```diff
@@ -0,0 +1,873 @@
+# Introduction
+
+## Scope
+
+The purpose of this document is to provide guidelines and conventions
+for the configuration and deployment of the Operating System Abstraction
+Layer (OSAL) to a desired platform or platforms. 
+
+## Background
+
+The goal OS Abstraction Layer is to promote the creation of portable and
+reusable real time embedded system software. Given the necessary OS
+abstraction layer implementations, the same embedded software should
+compile and run on a number of platforms ranging from spacecraft
+computer systems to desktop PCs. 
+
+## Applicable Documents
+
+| **Document ID** | **Document Title** |
+| --- | --- |
+| | |
+
+
+## Acronyms
+
+|   **Acronym**  |  **Description** |
+|---|---|
+| OS  | Operating System  |
+|  API | Application Programming Interface  |
+| CM  | Configuration Management  |  
+| CPU | Central Processing Unit  |  
+|  EEPROM |  Electrically Erasable Programmable Read-Only Memory |  
+|  HW, H/W | Hardware  |  
+|  RAM |  Random-Access Memory |  
+|  SW, S/W | Software |  
+|  TBD | To Be Determined |  
+
+## Glossary of Terms
+
+The following table defines the terms used throughout this document.
+These terms are identified as proper nouns and are capitalized. 
+
+| **Term**    |  **Definition**     |  
+|---|---|
+| Application  (APP)   |  A generic term for a computer program in a
```

Comment:
This table does not look like it is rendering correctly, at least when viewed on GitHub.
This is what it looks like when I view it.
<img width="556" alt="osal-markdown" src="https://user-images.githubusercontent.com/1873260/102124092-d264f880-3e15-11eb-8d8c-52c34ca2de5e.png">

Reply:
Updated several tables in terms of formatting. Should be all set to be marked as CCB-READY?